containers/notifications/Container.js

```diff
@@ -18,7 +18,7 @@ const NotificationsContainer = ({ notifications, removeNotification, hideNotific
         );
     });
 
-    return <div className="notifications-container flex flex-column flex-items-center">{list}</div>;
+    return <div className="notifications-container flex flex-column flex-items-center" data-cy="notifications-container">{list}</div>;
```

Comment:
Why do you use here `data-cy` and in the previous file `data-cy-appearance-conversations="single"` ? Isn't it better to always use ``data-cy` and add a scope inside the value itself ?

Reply:
The notifications container is global, in a sense that, it can be shown anywhere in the app.

The items with `data-cy-appearance-*` are for items shown only in `/settings/appearance`.